client/elements/suttaplex/sc-parallel-item.html

```diff
@@ -283,6 +283,10 @@
               return item.translated_title && item.original_title ? '' : 'hide-element';
           }
 
+          _removeSuttaText(item) {
```

Comment:
It seems this should be named `getTitleWithoutSuttaText` or something like that.